convert/harness/downgrader/downgrade.go

```diff
@@ -468,16 +468,21 @@ func (d *Downgrader) convertStepRun(src *v1.Step) *v0.Step {
 	for _, output := range spec_.Outputs {
 		outputs = append(outputs, convertOutput(output))
 	}
+	connectorRef := d.dockerhubConn
+	if spec_.Connector != "" {
```

Comment:
Should this not be the other way around? Default to the connector from the converted spec and override it if the user has provided a connector via the command line?

Reply:
The user-provided connector is used for everything. I think we can make it a bit more granular in the future. Keeping this for now